tests/ert3/console/integration/test_cli.py

```diff
@@ -63,6 +63,44 @@ def test_cli_init_subfolder(workspace):
             ert3.console._console._main()
 
 
+@pytest.mark.requires_ert_storage
+def test_cli_init_invalid_example(tmpdir, ert_storage):
+    with tmpdir.as_cwd():
+        args = ["ert3", "init", "--example", "something"]
+        with patch.object(sys, "argv", args):
+            with pytest.raises(
+                SystemExit, match="Example something is not a valid ert3 example."
+            ):
+                ert3.console.main()
+
+
+@pytest.mark.requires_ert_storage
+def test_cli_init_example(tmpdir, ert_storage):
+    with tmpdir.as_cwd():
+        args = ["ert3", "init", "--example", "polynomial"]
+        with patch.object(sys, "argv", args):
+            ert3.console.main()
+        polynomial_path = tmpdir / "polynomial"
+        if polynomial_path.exists() and polynomial_path.isdir():
+            # check that copied polynomial example is not an empty directory
+            assert len(polynomial_path.listdir()) > 0
+
+
+@pytest.mark.requires_ert_storage
+def test_cli_init_example_twice(tmpdir, ert_storage):
```

Comment:
Could you add a similar test where one creates a workspace, moves into it and then tries to create a second workspace?

Reply:
Currently if you do 
```
ert3 init --example spe1
cd spe1
ert3 init --example spe1
```
spe1 example will be copied inside spe1 but won't be initialised and you will see an error "Already inside an ert workspace." So this behavior is covered by `test_cli_init_subfolder` test

I can check whether we are already inside the folder with the same name as provided example name and that it has attributes of example (like stages.yml etc.), but I don't think I can check whether all files in the current working directory are identical to the one I am about to copy, and I am not sure we need that... or we do?

Reply:
I’m thinking the natural behavior would be to detect that it is already within a workspace and neither copy the files nor initialize. Otherwise, we carry out half the operation and then abandon 🤷🏻